source/digital_wallets/index.html.md.erb

```diff
@@ -68,14 +68,15 @@ To turn on Google Pay on a Worldpay service, you need to get a Google Pay mercha
 1. Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
 1. From **My Services**, select the service where you want to turn on Google Pay.
 1. Select **Settings**.
-1. Under **Payments**, select **Card payments**.
+1. Select **Payments**, then select **Card payments**.
```

Comment:
This will be correct until the new sidebar navigation is live, at which point there won't be a `settings` link,  just a settings heading in the sidebar nav with the individual settings links